pkg/dashboard/templates/image-scan-details.gohtml

```diff
@@ -3,70 +3,94 @@
+        <h3> Image vulnerabilities scan failed</h3>
+        <span> {{ .Description }} </span>
+      {{ end }}
 
-              <div class="name">
-                <span class="caret-expander"></span>
-                <span> {{ .Count }} issues with <strong> {{ .Severity }} </strong> severity </span>
-              </div>
+      {{ if (eq .ScanResult "NotFound") }}
+        <h3> No Image vulnerabilities scan data</h3>
+        <span>Image scanner might not support image OS or container registry</span>
+        <span> {{ .Description }} </span>
+      {{ end }}
 
-              <div class="expandable-content">
-                {{ range .CVEs }}
-                  <p> <i>{{ .Id }}</i> in <strong>{{ .PackageName }}</strong> version {{ .InstalledVersion }}</p>
-                  <p> <strong>Title:</strong> {{ .Title }} </p>
-                  <p> <strong>Remediation:</strong> Update package to {{ .FixedVersion }} </p>
-                  <p> {{ .Description }} </p>
-                  <ul>
-                    {{ range .References }}
-                      <li> <a href=">{{ . }}">{{ . }}</a> </li>
+      {{ if (eq .ScanResult "Succeeded") }}
+          {{ range $tIndex, $target := .ScanTargets }}
+            <ul>
+              <li>
+                <input type="checkbox" id="target{{ $tIndex }}" checked/>
+                <label for="target{{ $tIndex }}">
+                  Target: {{ $target.Name }}
+                </label>
+                <ul>
+                    {{ range $gIndex, $element := $target.VulnerabilitiesGroups }}
+                      <li>
+                        <input type="checkbox" id="t{{ $tIndex }}g{{ $gIndex }}" checked/>
+                        <label for="t{{ $tIndex }}g{{ $gIndex }}">
+                            {{ $element.Count }} issues with
+                          <strong class="severity-{{ $element.Severity }}"> {{ $element.Severity }} </strong>
+                          severity
+                        </label>
+                        <ul>
+                            {{ range $cveIndex, $cve := $element.CVEs }}
+                              <li>
+                                <input type="checkbox"
+                                       id="t{{ $tIndex }}vg{{ $gIndex }}cve{{ $cveIndex }}"/>
+                                <label for="t{{ $tIndex }}vg{{ $gIndex }}cve{{ $cveIndex }}">
+                                  <i>{{ $cve.Id }}</i> in <strong>{{ $cve.PackageName }}</strong>
+                                  version {{ $cve.InstalledVersion }}
+                                </label>
+                                <ul>
+                                  <li class="cve-detail">
+                                      {{ if gt (len $cve.Title) 0 }}
+                                        <p><strong>Title:</strong> {{ $cve.Title }} </p>
+                                      {{ end }}
+                                    <p><strong>Remediation:</strong> Update package
```

Comment:
Remember, `cve.FixedVersion` might be null or empty?

Maybe we have to add if-statement to display
> **Remediation**: Update package to {{ $cve.FixedVersion }}

only when value exists?

Or add empty 
> **Remediation**: 

if no data...

I'm opened to another proposals ;)

Reply:
best would be to check if cve.FixedVersion is empty, and display the remediation if we have a value.